HSchool-project/db.json

```diff
@@ -0,0 +1,18 @@
+{
+  "Course": [
+    {
+      "courseName": "angular",
+      "progress": 25,
+      "author": "john",
+      "courseNo": 1,
+      "id": 1
+    },
+    {
+      "courseName": "java",
+      "progress": 2,
+      "author": "peter",
+      "courseNo": 2,
+      "id": 2
+    }
+  ]
+}
```

Comment:
- [ ] You don't need to put the static data, let's have the db empty before hand, so we will rely on user's input